go/bundle/README.md

```diff
@@ -84,6 +84,13 @@ gen-bundle -dir static -baseURL https://example.com/ -o foo.wbn -primaryURL http
 If `-baseURL` flag is not specified, resources will have relative URLs in the generated bundle file.
 
 ### sign-bundle
+
+There are two supported ways to sign: using signed exchanges or integrity block, which can be separated using `-signType` `signaturessection` or `integrityblock` flag values. Without the signType flag, the default is `signaturessection`. 
+
+#### Using Signed Exchanges
```

Comment:
Would you replace "signed exchanges" in this doc with "signatures section", and add a link to `../../extensions/signatures-section.md`?